sw/device/tests/crypto/BUILD

```diff
@@ -237,9 +255,10 @@ autogen_cryptotest_header(
     tool = "//sw/device/tests/crypto:rsa_3072_verify_set_testvectors",
 )
 
-opentitan_functest(
+opentitan_test(
     name = "rsa_3072_verify_functest_wycheproof",
     srcs = ["rsa_3072_verify_functest.c"],
+    exec_env = EARLGREY_TEST_ENVS,
```

Comment:
This test is probably failing in the cw310 ROM tests because the whycheproof tests are way too big to be a ROM_EXT stage (ie: larger than 64K).

You probably have to manually specify the exec_envs here to omit `fpga_cw310_rom_with_fake_keys`.

Reply:
Done.